weblab/templates/datasets/includes/dataset_header.html

```diff
@@ -11,14 +11,15 @@ <h1 id="entityname">
   <small>Created <time>{{dataset.created_at}}</time> by <em>{{dataset.author.full_name}}</em>.
     {% can_delete_entity dataset as can_delete %}
     {% if can_delete %}
-      Delete dataset:
+
       <a title="delete dataset" href="{% url 'datasets:delete' dataset.id %}">
+          Delete dataset:
         <img src="{% static 'img/delete.png' %}" alt="delete dataset"
              title="delete dataset"/></a>
     {% endif %}
 
     <a title="Transfer ownership of dataset" href="{% url 'datasets:transfer' dataset.id %}">
-      Transfer ownership </a>
+    | Transfer ownership |</a>
```

Comment:
I'd display the first vertical bar within the if can_delete, otherwise it'll look a bit odd for users who can't delete the dataset.

Though it occurs to me the transfer and rename links also need to be within an if block, since only the owner should be allowed to do the first, and only someone with edit rights should be able to do the second. Or possibly anyone able to manage the dataset can do both, as with models/protocols in the file below?